src/AzureAuth/Commands/Ado/Pat/CommandScopes.cs

```diff
@@ -0,0 +1,47 @@
+
+using McMaster.Extensions.CommandLineUtils;
+
+using Microsoft.Authentication.AdoPat;
+using Microsoft.Extensions.Logging;
+
+using System;
+using System.Linq;
+
+namespace Microsoft.Authentication.AzureAuth.Commands.Ado.Pat
+{
+    /// <summary>
+    /// Command to print the list of avaialable scopes

```

Comment:
```suggestion
    /// Command to print the list of available scopes

```

Reply:
@kyle-rader-msft, I hope you don't mind, I just applied this small typo fix for you.